etc/ReplayOfflineConfiguration.py

```diff
@@ -1319,7 +1343,8 @@
 
 for dataset in DATASETS:
     addDataset(tier0Config, dataset,
-               do_reco=False,
+               do_reco=True,
+               alca_producers=["PPSCalTrackBasedSel", "PromptCalibProdPPSTimingCalib", "PromptCalibProdPPSAlignment"],
```

Comment:
This is analyzed at prompt-reco level right? so the `PromptCalibProd*` producer (which are aiming at PCL) don't make much sense here. They only work at express-reco level. Did I misunderstood?

Reply:
Fixed!